roadmap/implementers-guide/src/runtime/router.md

```diff
@@ -27,9 +110,108 @@ No initialization routine runs for this module.
 
 ## Routines
 
+The following routines are intended to be invoked by paras' upward messages.
+
+* `init_open_channel(recipient)`:
+  1. Check that the origin of this message is a para. We say that the origin of this message is the `sender`.
+  1. Check that the `sender` is not `recipient`.
+  1. Check that the `recipient` para exists.
+  1. Check that there is no existing open channel request from sender to recipient.
+  1. Check that the sum of the number of already opened HRMP channels by the `sender` (the size of the set found `HrmpEgressChannelsIndex` for `sender`) and the number of open requests by the `sender` (the value from `HrmpOpenChannelRequestCount` for `sender`) doesn't exceed the limit of channels (`config.hrmp_max_parachain_outbound_channels` or `config.hrmp_max_parathread_outbound_channels`) minus 1.
+  1. Reserve the deposit for the `sender` according to `config.hrmp_sender_deposit`
```

Comment:
@alistairstewart It would be helpful to have some more guidance on deposits for parachains/parathreads and how this interacts with the broader parachain / parathread deposits, respectively

Reply:
The easiest thing would be to have the sender always put down a fixed deposit. While it may be able to share code or storage location with the auction deposit, it shouldn't interact in any way.